index.html

```diff
@@ -1538,6 +1538,28 @@ <h4><dfn data-lt="version-time" data-dfn-type="dfn" id="version-time">version-ti
       </pre>
     </section>
 
+    <section>
+      <h4><dfn data-lt="relative-ref" data-dfn-type="dfn" id="relative-ref">relative-ref</dfn></h4>
+      <table class="simple" style="width: 100%;">
+        <thead>
+          <tr>
+            <th>Normative Definition</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>
+              <a href="http://w3c.github.io/did-core/#generic-did-url-parameters">DID Core</a>
+            </td>
+          </tr>
+        </tbody>
+      </table>
+
+      <pre class="example" title="Example of relative-ref parameter">
+did:example:123?service=files&relative-ref=%2Fmyresume%2Fdoc%3Fversion%3Dlatest%23intro
+      </pre>
+    </section>
+
     <section>
       <h4><dfn data-lt="initial-state" data-dfn-type="dfn" id="initial-state">initial-state</dfn></h4>
```

Comment:
Not for this PR, but we determined that initial-state is unnecessary during the last special topic call and it should probably be removed from the registry.